samples/tables.js

```diff
@@ -610,11 +562,11 @@ function extractTableToGCS(
     .catch(err => {
       console.error('ERROR:', err);
     });
-  // [END bigquery_extract_gcs]
+  // [END bigquery_extract_table]
 }
 
 function insertRowsAsStream(datasetId, tableId, rows, projectId) {
-  // [START bigquery_insert_stream]
+  // [START bigquery_table_insert_rows]
```

Comment:
I assume you'll be updating https://cloud.google.com/bigquery/docs/streaming-data-into-bigquery to use the new region tag?

Reply:
Yes